packages/wrangler/src/bundle-reporter.tsx

```diff
@@ -0,0 +1,42 @@
+import { readdirSync, statSync } from "fs";
+import { join } from "node:path";
+import { gzipSync } from "node:zlib";
+import { Box, Text, render, useStdout } from "ink";
+import * as React from "react";
+import type { DirectoryResult } from "tmp-promise";
+
+const dirSize = (dirPath: string) =>
+  readdirSync(dirPath)
+    .map((file) => statSync(join(dirPath, file)))
+    .reduce((acc, { size }) => acc + size, 0);
```

Comment:
does this work for nested directories, or does it only work on flat directories? and/or does it matter?

Reply:
I don't think it matters as is, however, no this code would not recursively go deeper into other directories.

Reply:
this might not be a guarantee